content/xr-dev/virtual-reality/zero-to-quest.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Zero To Quest - Meta XR SDK
+---
+
+# Zero to Quest Meta XR SDK
+
+## Overview
+Developing for virtual reality can be a daunting task, especially for beginners. This guide aims to simplify the process by walking you through the necessary steps to develop VR applications for Meta's Quest devices, using the [Meta Building Blocks SDK](https://assetstore.unity.com/packages/tools/integration/meta-xr-all-in-one-sdk-269657?srsltid=AfmBOopGcg_wSYbVGG4iWQu9dGdDVLT57kF8zgaTrSOuXcQgZE1cRQFo).
+
+## Setting Up Your Unity Project
+- **Create a New Unity Project:**
+  - Choose either Unity's Universal Render Pipeline (URP) or the Built-in Render Pipeline.
+  - Ensure that the Android Module is installed:
```

Comment:
"Android Build Module" - searchable keywords. 

For instructions on that, see this page on builds, with a link. Including these details is fine, separating is so we only have to update/edit one place.